packages/editor/src/client/panel/memberPlanEditPanel.tsx

```diff
@@ -249,6 +255,7 @@ export function MemberPlanEditPanel({id, onClose, onSave}: MemberPlanEditPanelPr
             <Form.Group>
               <Form.ControlLabel>{t('articleEditor.panels.tags')}</Form.ControlLabel>
               <TagPicker
+                disabled={isDisabled}
```

Comment:
instead of disabling each single form element, couldn't we frame the whole form with a <PermissionControl>? I'm afraid it's not sustainable nor maintainable.

Reply:
This was the existing form setup. The disabled property was already on the form elements and I just added to the elements that didn't have it. Can we fix that in a separate ticket?